index.html

```diff
@@ -574,10 +574,10 @@ <h4><dfn data-lt="serviceEndpoint" data-dfn-type="dfn" id="serviceEndpoint">serv
     </section>
 
     <section>
-      <h3>Verification methods</h3>
+      <h3>Verification relationships</h3>
       <p>
-These properties pertain to parameters that can be used to independently verify
-a proof according to a particular method.
+These properties indicate the purpose for which the verification method in the value can
+be used to verify the controller of the DID.
```

Comment:
See https://w3c.github.io/did-core/#dfn-verification-relationship

Reply:
@rhiaro reading this sentence again... I'm not sure its very clear... perhaps: 

"A DID document expresses the relationship between the DID subject and a verification method using a verification relationship." pulled directly from did core is better.

Reply:
Consider also, linking here: https://w3c.github.io/did-core/#dfn-verification-method

Reply:
Fixed in b83bf30c8853616e4c3aaa7a165bf83df1cd60a3.